MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -1155,29 +1158,78 @@ public ReadOnlySpan<byte> Invoke(ushort ordinal, bool offsetsOnly = false)
         }
 
         /// <summary>
-        ///     Sets up the Generic MajorBBS Btrieve Database for Use
+        ///     Sets up the Generic MajorBBS Btrieve Database for use
         /// </summary>
         private void SetupGENBB()
         {
-            //If it's already setup, bail
-            if (Module.Memory.TryGetVariablePointer("GENBB", out _))
-                return;
+            //Setup the Pointer to the Global GENBB Address -- ensuring each module is referencing the same GENBB Pointer & GENBB Processor
+            if (!_globalCache.TryGet<IntPtr16>("GENBB-POINTER", out var genbbPointer))
+            {
+                //Declare Pointers and Locations for GENBB Data
+                Module.Memory.AddSegment(0x1000);
+                genbbPointer = new IntPtr16(0x1000, 0x0); //GENBB Btrieve Struct
+                var genbbNamePointer = new IntPtr16(0x1000, 0x100); //GENBB File Name Pointer
+                var genbbDataPointer = new IntPtr16(0x1000, 0x200); //GENBB Record Pointer
```

Comment:
can you add some comments on why these values were chosen?

Reply:
I see down below VARIABLE_BASE_SEGMENT has the same value, use those constants instead?

Reply:
`VARIABLE_BASE_SEGMENT` is 0x100 vs. 0x1000

I chose them just to put them on a round segment far enough out from the variables declarations.